src/components/media.js

```diff
@@ -235,58 +240,56 @@
       idx === '0' ? '0rem' : style.getSpacing(idx, device);
     return {
       devWrapper: {
-        display: 'flex',
         '& > *': {
           pointerEvents: 'none',
         },
       },
-      hasContent: {
-        width: 'fit-content',
-        height: 'fit-content',
+      root: {
+        width: ({ options: { width } }) =>
+          !width || /^\d+$/.test(width.trim()) ? '100%' : width,
```

Comment:
Maybe move this to a constant to make it clearer what it does, and make it reusable
```typescript
const onlyContainsDigits = /^\d+$/.test(width.trim());
```